docs/content/en/docs/getting-started/production-environment/vsphere-getstarted.md

```diff
@@ -219,6 +224,21 @@ Follow these steps if you want to use your initial cluster to create and manage
 
    To add more workload clusters, go through the same steps for creating the initial workload, copying the config file to a new name (such as `eksa-w02-cluster.yaml`), modifying resource names, and running the create cluster command again.
 
+1. ###### Install curated packages:
+
+   [cert-manager](https://cert-manager.io/) installation is required before installing curated packages on workload clusters. You can install cert-manager using the command below, then complete the curated packages installation by following instructions [here]({{< relref "../../tasks/packages" >}}).
+
+   ```bash
+   helm repo add jetstack https://charts.jetstack.io
+   helm repo update
+   helm install \
```

Comment:
This should use the helm chart from our ECR repository. Make a note about helm 3.8+ or setting the appropriate experiemental environment variables

Reply:
I removed the instructions here to address the "redundancy" comment. Users will follow instructions under `tasks/packages` at one single place.